docs/en/source/API_walkthrough.rst

```diff
@@ -146,7 +146,7 @@ First, we are going to create house2:smartrooms:room1. Let's assume that at enti
 		"type": "Relationship",
 		"object": "smartcity:houses:house2"
 	  },
-	  "@context": [{"Room": "urn:mytypes:room", "temperature": "myuniqueuri:temperature", "isPartOf": "myuniqueuri:isPartOf"},"https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"]
```

Comment:
[Minor] why are you remove this @context. Please revert the changes same as previous.

Reply:
I have reverted the changes same as previous. Please review again